overlord/ifacestate/helpers.go

```diff
@@ -719,3 +726,41 @@ func ensureSystemSnapIsPresent(st *state.State) error {
 	_, err := snapstate.CoreInfo(st)
 	return err
 }
+
+func hotplugTaskSetAttrs(task *state.Task, deviceKey, ifaceName string) {
+	task.Set("device-key", deviceKey)
+	task.Set("interface", ifaceName)
+}
+
+func hotplugTaskGetAttrs(task *state.Task) (deviceKey, ifaceName string, err error) {
```

Comment:
The more natural order on the results here might be _ifaceName, deviceKey,_ since the device key is built by and dependent on the interface.